uni/lib/controller/local_storage/app_shared_preferences.dart

```diff
@@ -117,6 +119,17 @@ class AppSharedPreferences {
     return prefs.setInt(themeMode, (themeIndex + 1) % 3);
   }
 
+  static Future<void> setLocale(String appLocale) async {
+    final prefs = await SharedPreferences.getInstance();
+    await prefs.setString(locale, appLocale);
+  }
+
+  static Future<Locale> getLocale() async {
+    final prefs = await SharedPreferences.getInstance();
+    final appLocale = prefs.getString(locale) ?? Platform.localeName;
```

Comment:
What happens if we are e.g. in France? Perhaps we should have a strict relation between available locales and the ones we implement (and provide the default english)